src/core/thread/mle_router.cpp

```diff
@@ -535,6 +542,8 @@ Error MleRouter::SendLinkRequest(Neighbor *aNeighbor)
     TxMessage *          message;
     Ip6::Address         destination;
 
+    VerifyOrExit(mMulticastLinkRequestDelay == 0);
```

Comment:
I think while `mMulticastLinkRequestDelay > 0` we block any Link Request even unicast ones to a specific neighbor. May be good to call it `mLinkRequestDelay`?